README.md

```diff
@@ -147,19 +158,25 @@ domainlist relay_to_domains = ${lookup mysql{RELAY_DOMAINS}}
 hostlist   relay_from_hosts = localhost : @ : 192.168.0.0/24
 #trusted_users = www-data
 ```
-Specify here, the username and group under which Exim runs. This combination is also that under which mailman must run in order to work:
+These lines configure database connectivity. You need to uncomment one of them (depending on the database type you have chosen) and adjust it to match your setup. You have at least to change the word 'CHANGE' you used for the 'vexim' database user, which you have created before. The socket path depends on your system for Debian it is:
```

Comment:
Did you accidentally erase the words "to the password" ?
Also, please add a comma before "for Debian".

After this, I think you can merge. :)